src/utils.js

```diff
@@ -24,7 +24,7 @@ module.exports.getRequireLoader = function(filePath, chunkName, moduleName, inli
     'loadChildren: function() { return new Promise(function (resolve, reject) {',
     '  ' + (isJs ? 'require' : '(require as any)') + '.ensure([], function (' + (isJs ? 'require' : 'require: any') + ') {',
     '    resolve(' + requireString + ');',
-    '  }, function(e) {',
+    '  }, ' + (isJs ? 'e' : '(e: any)') + ' => {',
```

Comment:
I apologize if I wasn't clear.  Leave the function call as is, just use the check for the `e`.

```
'  }, function (' + (isJs ? 'e' : '(e: any)') + ') {',
```

Almost there!

Reply:
My bad, sorry about that. I'm on it. :wink: